### PR TITLE
Prepare for AttributeContainers other than Run

### PR DESCRIPTION
**Pull request by pkasprzyk**

### CODE REVIEW

File: neptune/new/types/state.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from enum import Enum
+
+
+class ContainerState(Enum):
```
Comment (aniezurawski) [inline neptune/new/types/state.py:19]: I think it can be internal.

File: neptune/new/attribute_container.py [hunk 2]
```diff
@@ -0,0 +1,461 @@
+#
```
Comment (aniezurawski) [inline neptune/new/attribute_container.py:1]: Can it be internal?

File: neptune/new/run.py [hunk 3]
```diff
@@ -141,397 +93,18 @@ def __init__(
...
-            >>> # this will print out all Atom attributes stored in run as a dict
-
-        You may also want to check `fetch docs page`_.
-
-        .. _fetch docs page:
-            https://docs.neptune.ai/api-reference/run#fetch
-        """
-        return self._get_root_handler().fetch()
-
     def ping(self):
```
Comment (aniezurawski) [inline neptune/new/run.py:104]: It should be moved to Container as well, shouldn't it?

**Review by aniezurawski — CHANGES_REQUESTED** [review]

File: neptune/new/attribute_container.py (hunk 2 above)
Comment (pkasprzyk) [reply neptune/new/attribute_container.py:1]: This was my initial thought, but it implements a lot of public methods (assign, fetch etc.) -- and I don't really want it hidden in internal class.

File: neptune/new/run.py (hunk 3 above)
Comment (pkasprzyk) [reply neptune/new/run.py:104]: We thought (with @Herudaio) that a generic `Container` doesn't need to be pinged -- I was under the impression that `Project` won't have an `active` parameter (and no `running_time`, I think?), so pinging doesn't make sense.

File: neptune/new/types/state.py (hunk 1 above)
Comment (pkasprzyk) [reply neptune/new/types/state.py:19]: Right, I'll move this.

File: neptune/new/run.py (hunk 3 above)
Comment (pkasprzyk) [reply neptune/new/run.py]: discussed offline -- `Project` **can** have `active` and `running_time` , so moving `ping` to `AttributeContainer`

**Review by aniezurawski — APPROVED** [review]

**Pull request merged**